---
layout: entry
title: "Protein structure and sequence re-analysis of 2019-nCoV genome refutes snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1"
link: "https://doi.org/10.1021/acs.jproteome.0c00129"
author:
- Zhang, Chengxin; Zheng, Wei; Huang, Xiaoqiang; Bell, Eric W.; Zhou, Xiaogen; Zhang, Yang

summary:
- "Using metagenomic samples from Manis javanica we assembled a draft genome of the 2019-nCoV-like coronavirus. The genome shows 73% coverage and 91% sequence identity to the genome. In particular, the alignment of the spike surface glycoprotein receptor binding domain revealed 4-fold more variations in the bat coronanavirus RaTG13 than those in the Manis coroniavirus compared to the 2019."

original:
- "As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we first analyzed two recent studies which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. The re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, present however clear evidences rebutting these conclusions. Next, using metagenomic samples from Manis javanica we assembled a draft genome of the 2019-nCoV-like coronavirus, which shows 73% coverage and 91% sequence identity to the 2019-nCoV genome. In particular, the alignments of the spike surface glycoprotein receptor binding domain revealed 4-fold more variations in the bat coronavirus RaTG13 than those in the Manis coronavirus compared to 2019-nCoV, suggesting the pangolin as a missing link in the transmission of 2019-nCoV from bats to human."
---

